doc/releases.md

```diff
@@ -553,8 +553,30 @@ formatting passes the lint rules on `master`.
 to promote the builds as the `SHASUMS256.txt` file needs to be signed with the
 same GPG key!**
 
-Use `tools/release.sh` to promote and sign the build. When run, it will perform
-the following actions:
+Use `tools/release.sh` to promote and sign the build. Before doing this, you'll
+need to ensure you've loaded the correct ssh key, or you'll see the following:
```

Comment:
```suggestion
need to ensure that you've loaded the correct ssh key, or you'll see the following:
```

Reply:
the former is perfectly grammatically valid - do you just think this is clearer?

Reply:
Yeah, Ig it would be a good addition.

Reply:
I would err on the side of avoiding an unnecessary adjective that doesn’t substantially enhance a message - it artificially complicates for no obvious benefit.